apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -69,6 +71,34 @@ export function renderLoginTypeAndSharingControls(sectionId) {
   );
 }
 
+export function renderSectionTable(sectionId, loginType) {
+  if (experiments.MANAGE_STUDENTS) {
+    const dataUrl = `/v2/sections/${sectionId}/students`;
+    const element = document.getElementById('student-table-react');
+
+    $.ajax({
+      method: 'GET',
+      url: dataUrl,
+      dataType: 'json'
+    }).done(sectionData => {
+      console.log(sectionData);
+      sectionData = sectionData.map((section) => {
+        return {
+          ...section,
+          loginType: loginType
+        };
+      });
+      ReactDOM.render(
+        <ManageStudentsTable
+          studentData={sectionData}
+          id={sectionId}
+          loginType={loginType}
```

Comment:
Why do we both need to glom loginType onto sectionData AND provide it here?

Reply:
I remember being told that formatters for react-tabular should be const, meaning they don't have access to props, only rowData, but the table itself also needs the loginType because it toggles what columns and text to show.

If the formatters can be part of the class, which now thinking about it I don't see why not, we could just use the info from the top level component. 

formatters now: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/manageStudents/ManageStudentsTable.jsx#L38

Reply:
If we were to leave it as is, we would also presumably want to add loginType to studentSectionDataPropType?